.pre-commit-config.yaml

```diff
@@ -1,8 +1,29 @@
+fail_fast: true
```

Comment:
I think this should be removed, since it would prevent a user from fixing all issues after a single attempt to create a commit. Thoughts?

Reply:
There are times when I would rather it fail fast so I can just fix each issue as it comes, but also times when I would rather see them all. The time's I prefer a fast fail though is usually just in testing, which isn't included here, so I think this is a good point, will remove